client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -2136,16 +2136,6 @@ Undocumented.prototype.getExport = function( siteId, exportId, fn ) {
 	}, fn );
 };
 
-Undocumented.prototype.timezones = function( params, fn ) {
-	if ( typeof params === 'function' ) {
-		fn = params;
-		params = {};
-	}
-
-	let query = Object.assign( {}, params, { apiNamespace: 'wpcom/v2' } );
-	return this.wpcom.req.get( '/timezones', query, fn );
-};
-
```

Comment:
And I need moar coffee since we shouldn't be removing this until there's a full implementation in place